File: algorithms/leetcode/easy/1480-running-sum-of-1d-array/python/solution.py
# 1480. Running Sum of 1d Array
# 解題日期: 13天前

from typing import List

class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        output = 0
        outputList = []
        for num in nums:
            output += num
            outputList.append(output)
        return outputList

File: algorithms/leetcode/easy/1480-running-sum-of-1d-array/kotlin/Solution.kt
// 1480. Running Sum of 1d Array
// 解題日期: 13天前

class Solution {
    fun runningSum(nums: IntArray): IntArray {
        val result = mutableListOf<Int>()
        var sum = 0
        for (num in nums) {
            sum += num
            result.add(sum)
        }
        return result.toIntArray()
    }
}
